ow_power_system/config/system.cfg

```diff
@@ -28,16 +28,21 @@ voltage_range: 0.1			        # volts
 # values from other planetary rovers, and some are just guesses.
 
 # Baseline values (watts) for constant power draws.
-baseline_power_camera: 0.1
-baseline_power_communications: 0.6
 baseline_power_computing: 0.1
 baseline_power_heating: 1.2
 baseline_power_science_instr: 0.6
 baseline_power_sample_handling: 0.6
 baseline_power_other: 2.2
 
+
 # Power coefficients for dynamically computed power draw.
-wattage_coefficient_lights: 0.3    # power draw (W) per light at max intensity
+power_lights_coefficient: 0.3       # power draw (W) per light at max intensity
+power_camera_controller: 1.0        # power draw of the camera controller
+power_camera_exposed_coefficient: 2.0  # power draw while a camera is exposing
+power_comms_active: 28.0           # power draw when comms is activated
```

Comment:
Let's get @chetankul 's opinion, since the values he's been proposing seem to be of a different scale.  Also, note the other values are per-cell based on a specific battery pack.  Personally, I think the total power draw of each component would be more intuitive.  Let's get @chetankul 's opinion on this, too.

Reply:
Will treat the stereocam as a single system.

Reply:
Will change CCU consumed value to 1.6W

Reply:
done